packages/core/src/builder.class.ts

```diff
@@ -2640,7 +2640,9 @@ export class Builder {
     const isApiCallForCodegenOrQuery = isApiCallForCodegen || this.apiEndpoint === 'query';
 
     if (this.apiEndpoint === 'content') {
-      queryParams.enrich = true;
+      queryParams.noTraverse = queue[0].limit !== 1;
+      queryParams.includeRefs = true;
+
```

Comment:
We should move this logic before the following code blocks:

-  https://github.com/BuilderIO/builder/blob/1908d27cb9e27aac831a451ae3a66a842252ffb1/packages/core/src/builder.class.ts#L2604-L2616
- https://github.com/BuilderIO/builder/blob/1908d27cb9e27aac831a451ae3a66a842252ffb1/packages/core/src/builder.class.ts#L2627-L2630

These are two ways customers can override properties, so they need to exist after you set the `noTraverse` and `includeRefs` defaults.


I also just noticed these two `noTraverse` settings: https://github.com/BuilderIO/builder/blob/1908d27cb9e27aac831a451ae3a66a842252ffb1/packages/core/src/builder.class.ts#L2495-L2497
https://github.com/BuilderIO/builder/blob/1908d27cb9e27aac831a451ae3a66a842252ffb1/packages/core/src/builder.class.ts#L2863-L2867

This means that `noTraverse` is already handled (set to `true` in `getAll()` which has `limit > 1`. So you don't need to set `noTraverse`, you only need to add the `includeRefs = true`.

TLDR:
- remove `noTraverse` default
- move the `includeRefs` default higher up so it can still be overridden by customer
- add some tests that show:
  - default value of `includeRefs` is `true`
  - if customer provides `includeRefs: false` to `builder.get()` or `builder.getAll()`, then they are able to update the URL